src/mlpack/tests/gmm_test.cpp

```diff
@@ -855,7 +858,8 @@ BOOST_AUTO_TEST_CASE(DiagonalGMMTrainEMOneGaussian)
 
     arma::vec actualMean = arma::mean(data, 1);
     arma::vec actualCovar = arma::diagvec(
-        arma::ccov(data, 1 /* biased estimator */));
+        mlpack::math::ColumnCovariance(data,
+        arma::size_t(1) /* biased estimator */));
```

Comment:
`arma::size_t` ? did you mean `std::size_t` ?

Reply:
@conradsnicta , Sorry for the big blunder , Shouldn't have used sublime feature :)